packages/grid/src/elements/pane/PaneProvider.spec.tsx

```diff
@@ -216,6 +216,7 @@ describe('PaneProvider', () => {
           "getGridTemplateColumns": [Function],
           "getGridTemplateRows": [Function],
           "getRowValue": [Function],
+          "id": undefined,
```

Comment:
Seems like there should be tests added here to suss out the new features of a nested pane provider.

Reply:
Fixed in https://github.com/zendeskgarden/react-components/pull/1356/commits/7cfdaf949e0ff030eb133e4ac672aa2403c64b0b